Extensions/Xtensive.Orm.Localization.Tests/ProjectionToCustomTypesTests.cs

```diff
@@ -0,0 +1,55 @@
+using System.Linq;
+using System.Threading;
+using NUnit.Framework;
+using Xtensive.Orm.Localization.Tests.Model;
+
+namespace Xtensive.Orm.Localization.Tests
+{
+  [TestFixture]
+  public class ProjectionToCustomTypesTests : AutoBuildTest
+  {
+    protected override void PopulateDatabase()
+    {
+      using (var session = Domain.OpenSession()) {
+        using (var ts = session.OpenTransaction()) {
+
+          // populating database
+          var m1 = new Country(session) {
+            Identifier = "HUN",
+            Name = "Magyarország"
+          };
+          var m2 = new Country(session) {
+            Identifier = "RUS",
+            Name = "Oroszország"
+          };
+          using (new LocalizationScope(EnglishCulture))
+            m2.Name = "Russia";
+          using (new LocalizationScope(SpanishCulture))
+            m2.Name = "Rusia";
+
+          ts.Complete();
+        }
+      }
+    }
+
+    [Test]
+    public void EntityHierarchyWithAbstractPropertyTest()
+    {
+      Thread.CurrentThread.CurrentCulture = EnglishCulture;
+      using (var session = Domain.OpenSession()) {
+        using (var ts = session.OpenTransaction()) {
```

Comment:
Same extra indent for ```using``` chain. I mentioned it in another comment up above.

Reply:
done